Classes/Command/AssetCommandController.php

```diff
@@ -22,20 +21,22 @@ class AssetCommandController extends CommandController
 
     /**
      * @Flow\Inject
-     * @var AssetRepository
      */
-    protected $assetRepository;
+    protected AssetRepository $assetRepository;
 
     /**
+     * Takes care of exporting all assets stored in Neos as files. This can be very helpful for backups.
+     *
      * @param string $targetPath The path where the resource files should be exported to.
      * @param bool $emptyTargetPath If set, all files in the target path will be deleted before the export runs
      * @throws FilesException
+     * @throws StopCommandException
      */
     public function exportCommand(string $targetPath, bool $emptyTargetPath = false): void
     {
         if (!is_dir($targetPath)) {
             $this->outputLine('The target path does not exist.');
-            exit(1);
+            $this->quit(1);
```

Comment:
Description of the `quit()` function:

```
    /**
     * Exits the CLI through the dispatcher and makes sure that Flow is properly shut down.
     *
     * If your command relies on functionality which is triggered through the Bootstrap
     * shutdown (such as the persistence framework), you must use quit() instead of exit().
     *
     * @param integer $exitCode Exit code to return on exit (see http://www.php.net/exit)
     * @throws StopCommandException
     * @return void
     */
```

https://github.com/neos/flow-development-collection/blob/5f36937b946e03ea47997402acf50bc47229b8b6/Neos.Flow/Classes/Cli/CommandController.php#L327